power-control-x86/src/power_control.cpp

```diff
@@ -614,50 +614,48 @@ static void powerRestorePolicyCheck()
                 const std::variant<std::string>& restorepolicy) {
         if (ec)
         {
+            std::cerr << "ec Error in power restore policy check\n";
             return;
         }
         const std::string* policy =
                 std::get_if<std::string>(&restorepolicy);
         if (policy == nullptr)
         {
-            std::cerr << "Unable to read restore policy\n";
+            std::cerr << "Unable to read power restore policy\n";
             return;
         }
 
-        if (policy->compare("None") == 0)
```

Comment:
We still need this statement.

Reply:
However, the condition itself may need to initialized to proper value.

Reply:
It is now moved to the end of if else statement as the last else to catch all other conditions

        else if (policy->compare("xyz.openbmc_project.Control.Power.RestorePolicy.Policy.AlwaysOff") == 0)
            std::cerr << "Restore policy to Always Off\n";
        else
            std::cerr << "Restore policy to None\n";